### DISCUSSION TITLE
ValidationRule.IsEqual Usage...

### DISCUSSION THREAD

**Discussion by madcoda9000** [Megabit/Blazorise#4127] [Q&A] [answered]
Hello,

is there any example available on how to use ValidationRule.IsEqual?

I want to compare to password fields.

Thx
Sascha

**Comment by stsrki** [comment, marked as answer]
I believe `ValidationRule.IsEqual` is an obsolete API, and we should remove it. 

For comparing password look at our demo as an example https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/ValidationsPage.razor#L52

**Comment by madcoda9000** [comment]
Hello,

many thx for your fast answer. Yes i was able to validate and compare with own functions as you did in your example. But i wonder if there is a isEqual Method in the Validator object, why is it not useable like:

Validator.isEqual(MyModel.password, MyModel.ConfirmPassword)
To me it seems to be great. It would be sad if this would become obsolete :-(

**Reply by stsrki** [reply]
The problem is delegate signature is not the same as other functions since it needs to receive a value to compare. We would have to figure out how to handle that scenario.

**Reply by madcoda9000** [reply]
ok, thank you!
